src/main/java/com/jcabi/github/Github.java

```diff
@@ -90,6 +90,12 @@
 public interface Github {
 
     /**
+     * Root Github Repository path.
+     * @return Root repo path
+     */
+    String rootRepoPath();
```

Comment:
@kkashi01 This is not correct, I'm afraid. This method has nothing to do with ``Github``. Interface ``Github`` should only contain methods that are "entry points" to their corresponding APIs (repos, issues, comments etc). 

From our discussion, I had imagined the following change (e.g. for class ``RtRepos``): 

Constructor of ``RtRepos`` now:

```java
RtRepos(final Github github, final Request req) {
    this.ghub = github;
    this.entry = req;
}
```
Constructor of ``RtRepos`` after:

```java
RtRepos(final Github github, final Request req) {
    this.ghub = github;
    this.entry = req.uri().path("/repos").back;
}
```

You see, we added the ``/repos`` path element in the ctor, so we don't have to hardcode it throughout the methods of this class.

Maybe it will be a little more complex because some methods of the Repos API don't have the ``/repos`` prefix -- some have ``/user/repos``, or ``/orgs/repos``.